cfgov/ask_cfpb/views.py

```diff
@@ -93,12 +97,17 @@ def ask_search(request, language='en', as_json=False):
         results_page.result_query = ''
         return results_page.serve(request)
 
-    search = AskSearch(search_term=search_term, language=language)
-    if search.queryset.count() == 0:
-        search.suggest(request=request)
+    if flag_enabled('ELASTIC_SEARCH_DSL'):
```

Comment:
It looks like you're using the flag in `urls.py` to switch view functions based on it. I don't believe this is doing anything, and it would probably be cleaner to use `urls.py` to switch whole views instead of checking the flag state multiple times within this view.

Reply:
Removed flag_enabled from ask_cfpb views.py

Reply:
Removed flag_enabled from ask_cfpb views.py as flag is already being used in urly.py.